README.md

```diff
@@ -115,15 +117,38 @@ By default, the actions installs (if necessary) and then uses the `stable` toolc
 If you want to setup the toolchain manually, you can set `rust-toolchain` to `manual`. In this case, the action assumes some Rust toolchain is already installed and uses the default one:
 ```yaml
 - name: Setup Rust
-  uses: dtolnay/rust-toolchain@master
+  uses: actions-rust-lang/setup-rust-toolchain@v1
   with:
-    toolchain: stable 2 months ago
+    toolchain: stable
 - name: Check semver
   uses: obi1kenobi/cargo-semver-checks-action@v2
   with:
     rust-toolchain: manual
 ```
 
+### Build for target other than default
+
+By default, the action uses the default target (based on the host platform). You can force using a different target (which will be installed if missing) using `rust-target`:
```

Comment:
This looks good. The one remaining suggestion I have is to reinforce in the text that this example is written assuming the default Linux GitHub Actions runner, not a macOS aarch64 runner. That way, readers will understand _why_ they would use this option, not just _how_ to use it.

One way to do that would be something like the following:
```suggestion
By default, the action uses the default target based on the host platform. You can force using a different target (which will be installed if missing) using `rust-target`. For example, you can check an `aarch64-apple-darwin` build while using the default Linux-based GitHub Actions runner:
```

Reply:
Sounds good!